test/e2e/pill.test.js

```diff
@@ -9,7 +8,50 @@ describe('vl-pill', async () => {
         return vlPillPage.load();
     });
 
-    it("Dummy test om de browsers te doen sluiten", () => {
-    	assert.isTrue(true);
+    it("Als gebruiker kan ik de inhoud van een pill zien", async () => {
+        const pill = await vlPillPage.getStandardPill();
+        const content = await pill.getContent();
+        await assert.equal(content[0].textContent.trim(), 'Optie 1');
```

Comment:
Niet beter innerText gebruiken zodat je niet moet trimmen?

Reply:
Gaan we toch niet doen, van de text kunnen we geen VlElement/WebElement maken, waardoor we geen getText() kunnen aanroepen. innerText is in dit geval ook geen optie.